diagnostic.md

```diff
@@ -3,36 +3,40 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+1) Fast
+2) Can have "back" button / bookmark, better ui overall.
+3) Data-binding
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+Handlebars...unless I am reading wrong...but you can add some more stuff. (I.e. components)
```

Comment:
👍 
Yep, it's handlebars with some extra helpers and functionality (like linking to components).